requirements/debug

```diff
@@ -0,0 +1,5 @@
+# These are dependencies to help with debug and profiling, so we always want
+# the latest versions.
+
+# Interactive console on steroids (even makes your coffee)!
```

Comment:
I'm not at ease with this tongue-in-cheek comment. If this is added as a dependency, I'd prefer that an actual justification along with a link to the documentation is added as a comment 🙂